test/verify/check-networking-firewall

```diff
@@ -240,9 +246,12 @@ class TestFirewall(NetworkCase):
         # firewalld > 0.7.0 supports including services into another service
         if m.image in ['rhel-8-3', 'rhel-8-3-distropkg', 'debian-testing', 'fedora-testing', "fedora-32", "fedora-33"]:
             addService('public', 'freeipa-4')
-            b.click(".zone-section[data-id='public'] tr[data-row-id='freeipa-4']")
-            b.wait_present(".zone-section[data-id='public'] tbody.open .ct-listing-panel:contains(Included Services)")
-            b.click(".zone-section[data-id='public'] tr[data-row-id='freeipa-4']")
+            b.click(".zone-section[data-id='public'] tr[data-row-id='freeipa-4'] button")
+            if m.image not in ["rhel-8-3-distropkg"]: # Changed in #14597
```

Comment:
We won't need this anymore :) @martinpitt is removing these cases in https://github.com/cockpit-project/cockpit/pull/14608

Reply:
So shall I remove all distropkg conditions that I introduced here?

Reply:
That landed now, so just in case you need another force-push, you can remove it. Otherwise, we'll clean it up in time when we introduce rhel-8-4-distropkg.